docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,349 @@
+    def_bool $(shields_list_contains,my_board_right)
+```
+</TabItem>
+<TabItem value="unibody">
+We will add a dongle to the unibody keyboard configuration. 
+
+```kconfig title="Kconfig.shield"
+config SHIELD_MY_BOARD_DONGLE
+    def_bool $(shields_list_contains,my_board_dongle)
+
+config SHIELD_MY_BOARD
+    def_bool $(shields_list_contains,my_board)
+
+```
+</TabItem>
+</Tabs>
+
+### Keyboard Defconfig
+
+:::info
+In the shield defconfig we will assume that the user does not want to use the dongle by default.
+This is done so that the user can easily enable the dongle in the user configuration without having to modify the shield configuration.
+:::
+
+<Tabs
+defaultValue="split"
+values={[
+{label: 'Split Keyboard', value: 'split'},
+{label: 'Unibody Keyboard', value: 'unibody'},
+]}
+>
+
+<TabItem value="split">
+
+Next we will define the roles of the keyboard halves. The left and right halves will be set as peripherals, and the dongle will be set as the central.
+We will also only give the central device the keyboard name.
+
+There is also an option to still use the left half as the central, and the right half as the peripheral. In that case the dongle will not be used.
+
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_BOARD_DONGLE
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT_BLE_CENTRAL_PERIPHERALS
+        default 2
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+
+if SHIELD_MY_BOARD_LEFT
+
+    # Setting this to y will make the left half the central
+    # We want it to be the central when dongle is not used
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    # Check if the left half is the central
+    if ZMK_SPLIT_ROLE_CENTRAL
+        config ZMK_KEYBOARD_NAME
+            default "My Board"
+            
+    endif # ZMK_SPLIT_ROLE_CENTRAL
+
+endif # SHIELD_MY_BOARD_LEFT
+
+if SHIELD_MY_BOARD_DONGLE || SHIELD_MY_BOARD_LEFT || SHIELD_MY_BOARD_RIGHT
+
+    config ZMK_SPLIT
+        default y
+
+endif
+```
+
+</TabItem>
+<TabItem value="unibody">
+
+Now we will add the dongle configuration to the unibody keyboard.
+We will default the unibody keyboard to not use the dongle configuration.
+
+```kconfig title="Kconfig.defconfig"
+if SHIELD_MY_BOARD_DONGLE
+
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+
+    config ZMK_SPLIT_ROLE_CENTRAL
+        default y
+
+    config ZMK_SPLIT
+        default y
+
+    # Prevent the dongle from going to sleep
+    config ZMK_SLEEP
+        default n
+
+    # Increase the transmit power of the dongle
+    config BT_CLTR_TX_PWR_PLUS_8
+        default y
+
+endif
+
+if SHIELD_MY_BOARD
+    config ZMK_KEYBOARD_NAME
+        default "My Board"
+    
+    config ZMK_SPLIT
+        default n
+    
+endif
+```
+
+</TabItem>
+</Tabs>
```

Comment:
Reading through this I think you could also get rid of the tabs from here too. Again, I'd try to phrase it in such a way that makes it clear that we are *adding/modifying* lines. To that extent I'd split this into three parts: One where you write the stuff that's in the dongle if block, one for editing the former central, and one for handling ZMK_SPLIT